tests/integration/transformers/test_base.py

```diff
@@ -129,3 +129,190 @@ def _reverse_transform(self, data):
     })
     pd.testing.assert_frame_equal(expected_transform, transformed)
     pd.testing.assert_frame_equal(reverse, data)
+
+
+def test_multi_column_transformer_same_number_of_columns_input_output():
+    """Test a multi-column transformer when the same of input and output columns."""
+    # Setup
+    class AdditionTransformer(BaseMultiColumnTransformer):
```

Comment:
I think we should actually test this example from the `HyperTransformer` and make sure it really works end to end